### ISSUE TITLE
Doesn't build anymore with Qt 5.15

### ISSUE THREAD

**Issue by adriweb** [KDAB/KDMacTouchBar#3]
This is due to https://github.com/qt/qtbase/commit/bee3d0fc1310da6685dce9cf9b1334fb8295aa9a which removes `NSImage *qt_mac_create_nsimage(const QIcon &icon, int defaultSize = 0)`.

Looks like the class method `imageFromQIcon` can be used instead. But only if the current Qt version is >= 5.15.

**Comment by adriweb** [comment]
A patch to make it work would be to add the following, right after `NSImage *qt_mac_create_nsimage(const QIcon &icon, int defaultSize)` in https://github.com/KDAB/KDMacTouchBar/blob/34efc6352/src/kdmactouchbar.mm#L40: 

```cpp
#if QT_VERSION >= QT_VERSION_CHECK(5, 15, 0)
    @interface NSImage (QtExtras)
    + (instancetype)imageFromQIcon:(const QT_PREPEND_NAMESPACE(QIcon) &)icon withSize:(int)size;
    @end
    NSImage *qt_mac_create_nsimage(const QIcon &icon, int defaultSize)
    {
        return [NSImage imageFromQIcon:icon withSize:defaultSize];
    }
#endif
```

**Comment by cschleifenbaum** [comment]
Thank you for pointing this out. I just released v1.1.1 fixing the issue!

**Comment by adriweb** [comment]
Thanks :) Updating the project using it to a7db88b7b9f78a5c890c8699f1cd70b225d70872 :)

**Comment by adriweb** [comment]
Turns out there was a syntax error, I fixed that in PR #4.
